community/gvfs/APKBUILD

```diff
@@ -1,66 +1,57 @@
 # Contributor: Natanael Copa <ncopa@alpinelinux.org>
-# Maintainer: Natanael Copa <ncopa@alpinelinux.org>
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
 pkgname=gvfs
-pkgver=1.36.2
-pkgrel=1
+pkgver=1.40.1
+pkgrel=0
 pkgdesc="Backends for the gio framework in GLib"
-url="http://ftp.gnome.org/pub/gnome/sources/gvfs/${pkgver%.*}/"
+url="https://wiki.gnome.org/Projects/gvfs"
 arch="all"
-license="GPL"
-depends=""
-options="!check"  # no tests available
-triggers="$pkgname.trigger=/usr/lib/gvfs"
```

Comment:
Why is this trigger no longer useful?

Reply:
From the commit msg:

* Remove trigger which is likely to kill gvfs in mid-operation.

Without the trigger newly added gvfs-* packages won't be registered before a restart, but IMHO that's way better than just killing gvfs, which could happen while its in use.

Reply:
Presumably `gvfs` has a signal handler for `SIGUSR1` that does something intelligent about loading the files without just falling over in the middle of doing something.

Reply:
Alrighty then

Reply:
- [ ] add `triggers=` line again